### PR TITLE
chore: prepare 3.6 release article

### PR DESCRIPTION
**Pull request by markwhitfeld**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[X] Other... Please describe: Article
```

### CODE REVIEW

File: publication/2019-10-21_announcing_ngxs_3_6/article.md [hunk 1]
```diff
@@ -4,9 +4,11 @@
 
 ## Overview
 
+- 🦄 Ivy Support (TODO decide icon: 🦄 🌼 🍒 🌸 🎄 👀 🍀 🌿 🌈 🚀 ☘️ 🌟 ⭐️ 🎋 🏆)
```
Comment (splincode) [inline publication/2019-10-21_announcing_ngxs_3_6/article.md]: 🍀- use it! because official

image

Comment (markwhitfeld) [reply publication/2019-10-21_announcing_ngxs_3_6/article.md]: I do really like the Unicorn!
Ivy has been a mythical creature for such a long time!

Comment (arturovt) [reply publication/2019-10-21_announcing_ngxs_3_6/article.md]: When disputes about emojis begin, it seems to me that it’s better not to put emoji at all.... :rofl: :rofl:

Comment (markwhitfeld) [reply publication/2019-10-21_announcing_ngxs_3_6/article.md]: Pick a response you like

**Pull request merged**